lib/mzgl/midi/Midi.cpp

```diff
@@ -6,7 +6,13 @@
 //  Copyright © 2018 Marek Bereza. All rights reserved.
 //
 
-#include "Midi.h"
+#include <mzgl/midi/Midi.h>
+#ifdef __APPLE__
+#	include <TargetConditionals.h>
+#endif
+
+#if (TARGET_OS_MAC && !TARGET_OS_IOS) || defined(__linux__)
```

Comment:
RtMidi is used by windows too - also not sure but I think __linux__ will be defined on android maybe